### PR TITLE
Update zh_cn.json 

### PR DESCRIPTION
**Pull request by Cactusstudent**
Dont merge Please!
I still have some translation to correct.

### CODE REVIEW

**Comment by Cactusstudent** [comment]
A Question, is Scepter in wand_type means Copper Cap? if not, please tell me how this mean, please!!!

File: src/main/resources/assets/thaumcraft/lang/zh_cn.json [hunk 1]
```diff
@@ -11,9 +11,9 @@
 
   "advancements.journey.root.title": "§5神秘时代",
   "advancements.journey.root.description": "让我们来制作一把法杖！但先要做一个工作台",
```
Comment (Cactusstudent) [inline src/main/resources/assets/thaumcraft/lang/zh_cn.json:13]:
```suggestion
  "advancements.journey.root.description": "这将是一次美妙的旅途",
```

File: src/main/resources/assets/thaumcraft/lang/zh_cn.json [hunk 2]
```diff
@@ -11,9 +11,9 @@
...
-  "advancements.journey.paper_gathering.title": "研究...",
-  "advancements.journey.paper_gathering.description": "...需要大量的纸",
-  "advancements.journey.scribing_tools.title": "作划时间...",
+  "advancements.journey.paper_gathering.title": "研究……",
+  "advancements.journey.paper_gathering.description": "……需要大量的纸",
+  "advancements.journey.scribing_tools.title": "作划时间……",
   "advancements.journey.scribing_tools.description": "来整点空瓶子，用羽毛和墨囊就好",
```
Comment (Cactusstudent) [inline src/main/resources/assets/thaumcraft/lang/zh_cn.json:17]:
```suggestion
  "advancements.journey.scribing_tools.description": "用空瓶子、羽毛和墨囊合成笔与墨",
```

**Comment by JieningYu** [comment]
You can convert this pr to draft.

**Comment by Cactusstudent** [comment]
> You can convert this pr to draft.

OK

File: src/main/resources/assets/thaumcraft/lang/zh_cn.json [hunk 3]
```diff
@@ -10,11 +10,11 @@
...
   "advancements.journey.root.title": "§5神秘时代",
-  "advancements.journey.root.description": "让我们来制作一把法杖！但先要做一个工作台",
-  "advancements.journey.paper_gathering.title": "研究...",
-  "advancements.journey.paper_gathering.description": "...需要大量的纸",
-  "advancements.journey.scribing_tools.title": "作划时间...",
-  "advancements.journey.scribing_tools.description": "来整点空瓶子，用羽毛和墨囊就好",
+  "advancements.journey.root.description": "这将是一次美妙的旅途",
+  "advancements.journey.paper_gathering.title": "研究……",
+  "advancements.journey.paper_gathering.description": "……需要大量的纸",
+  "advancements.journey.scribing_tools.title": "作划时间……",
```
Comment (Cactusstudent) [inline src/main/resources/assets/thaumcraft/lang/zh_cn.json:16]:
```suggestion
  "advancements.journey.scribing_tools.title": "撰写时间……",
```

File: src/main/resources/assets/thaumcraft/lang/zh_cn.json [hunk 4]
```diff
@@ -110,28 +110,28 @@
...
+  "wand_rod.thaumcraft.wood": "木",
+  "wand_rod.thaumcraft.blaze": "烈焰",
+  "wand_rod.thaumcraft.bone": "骨",
+  "wand_rod.thaumcraft.greatwood": "宏伟之木",
+  "wand_rod.thaumcraft.ice": "冰",
+  "wand_rod.thaumcraft.obsidian": "黑曜石",
+  "wand_rod.thaumcraft.quartz": "石英",
+  "wand_rod.thaumcraft.reed": "甘蔗",
+  "wand_rod.thaumcraft.silverwood": "银树",
+  "wand_rod.thaumcraft.primal": "原始",
```
Comment (Cactusstudent) [inline src/main/resources/assets/thaumcraft/lang/zh_cn.json:133]:
```suggestion
  "wand_rod.thaumcraft.primal": "元始",
```

**Comment by Cactusstudent** [comment]
OK I have finished this

**Comment by Archasmiel** [comment]
Thank you again! Yesterday I added in README all who helps with mod translations.

**Comment by Archasmiel** [comment]
> A Question, is Scepter in wand_type means Copper Cap? if not, please tell me how this mean, please!!!

**This exact fields represent wand item type (as far as we have wand, scepter and staff in original TC 4):**
```
"wand_type.thaumcraft.wand": "Wand"
"wand_type.thaumcraft.scepter": "Scepter"
"wand_type.thaumcraft.staff": "Staff"
```

This fields (wand_cap, wand_rod, wand_type) used for name generation for different languages.
Dynamically by ARRP, mod reads translations and then registers generated name in same language.

For example English translation of scepter item, made by adding stick and iron caps:
**"Iron capped"(wand_cap) + " " + "Wood"(wand_rod) + " " + "Scepter"(wand_type)        =>     "Iron capped Wood Scepter"**

**Comment by Cactusstudent** [comment]
> made by adding stick and iron caps:

OK, I know. Maybe my translation is true. I will check all the translation and correct some of this in next version.

**Pull request merged**